### PR TITLE
Rails 4 Support

### PR DESCRIPTION
**Pull request by irubnich**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,4 +2,6 @@ sudo: false
 language: ruby
 rvm:
   - 2.0.0
-script: bundle exec rspec spec
\ No newline at end of file
+  - 2.1.7
+  - 2.2.2
+script: bundle exec rspec spec
```
Comment (pjambet) [inline .travis.yml]: would be great to add rails 3.2.x and 4.2.x (and maybe 4.0.x/4.1.x who knows?!) to the test matrix

**Comment by pjambet** [comment]
Looks good! Just one comment about the travis config.

File: lib/split_cacheable.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 require "split_cacheable/version"
 require 'split_cacheable/helper'
-require 'split_cacheable/engine' if defined?(Rails) && Rails::VERSION::MAJOR === 3
+require 'split_cacheable/engine' if defined?(Rails) && [3, 4].include?(Rails::VERSION::MAJOR)
```
Comment (irubnich) [inline lib/split_cacheable.rb]: This must only be 4

**Comment by irubnich** [comment]
@pjambet I adjusted this so it supports both 3.2 and rails 4 and updated the readme with the info. It's just the difference of adding or not adding the Rails 4 gem.

Version bump is now only to 1.1.0.

File: lib/split_cacheable.rb [hunk 3]
```diff
@@ -1,6 +1,8 @@
 require "split_cacheable/version"
 require 'split_cacheable/helper'
-require 'split_cacheable/engine' if defined?(Rails) && Rails::VERSION::MAJOR === 3
+if defined?(Rails) && [3, 4].include?(Rails::VERSION::MAJOR) && ActionController::Base.method_defined?(:fragment_cache_key)
```
Comment (irubnich) [inline lib/split_cacheable.rb]: @pjambet this check should completely ensure that split_cacheable has what it needs

File: README.md [hunk 4]
```diff
@@ -16,13 +16,23 @@ Then run:
 
     bundle install
 
+### Rails 4
+
+Because this gem uses action caching, a feature removed from Rails 4, you will also need to include the `protected_attributes` gem in your Gemfile:
+
+    gem 'protected_attributes'
```
Comment (irubnich) [inline README.md]: Wrong gem, I'm an idiot

**Comment by irubnich** [comment]
@pjambet plz

**Comment by pjambet** [comment]
:+1:

**Comment by irubnich** [comment]
@danielschwartz can you give me push permissions for this gem on rubygems? Would love to push out this new version (not urgent)

**Pull request merged**